modules/persistent-storage-csi-efs-metrics-procedure-gui.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+:_mod-docs-content-type: PROCEDURE
+[id="efs-metrics-procedure-gui_{context}"]
+= Enabling usage metrics using the web console
+
+To enable Amazon Web Services (AWS) Elastic File Service (EFS) Storage Container Storage Interface (CSI) usage metrics using the web console:
+
+. Click *Administration* > *CustomResourceDefinitions*.
+
+. On the *CustomResourceDefinitions* page next to the *Name* dropdown box, type `clustercsidriver`.
+
+. Click *CRD ClusterCSIDriver*.
+
+. Click the *YAML* tab.
+
+. Under `spec.aws.efsVolumeMetrics.state`, change the value from `Disabled` (default) to `RecursiveWalk`.
```

Comment:
Same comment, remove "change the value from `Disabled`" because by default this path doesn't exist.

Reply:
Got it.